src/plugins/recordTypes/deaccession/fields.js

```diff
@@ -0,0 +1,42 @@
+export default (configContext) => {
+  const {
+    AutocompleteInput,
+  } = configContext.inputComponents;
+
+  const {
+    configKey: config,
+  } = configContext.configHelpers;
+
+  return {
+    document: {
+      'ns2:deaccessions_common': {
+        deaccessionApprovalGroupList: {
+          deaccessionApprovalGroup: {
+            deaccessionApprovalIndividual: {
+              [config]: {
+                view: {
+                  type: AutocompleteInput,
```

Comment:
@kspurgin Just confirmed that we don't need to re-declare the type here, so this can be removed (and the import/other places). If you want I can go through and remove it, lmk.

Reply:
@mikejritter - New commit removing type declarations pushed just now